react/react1/week3/to-do-app/src/components/CurrentDate.jsx

```diff
@@ -0,0 +1,7 @@
+
+    var now = new Date();
+    var day = ("0" + now.getDate()).slice(-2);
+    var month = ("0" + (now.getMonth() + 1)).slice(-2);
+    var today = now.getFullYear() + "-" + month + "-" + day;
+
```

Comment:
I used something like this `const currentDate = new Date();
  const currentDateHours = currentDate.setHours(0, 0, 0, 0);`

Reply:
and then compare with other date also with setHours(0, 0, 0, 0)